definitions/variable/emissions/emissions.yaml

```diff
@@ -43,28 +46,33 @@
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+- Emissions|*|AFOLU|Biomass Burning:
+    description: Agricultural Waste Burning (DESCRIPTION TO BE UPDATED)
+    unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land:
     description: Emissions of {Level-3 Species} from forestry and other land use
     unit: "{Level-3 Species}"
-- Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions of {Level-3 Species} from managed peatlands (drained and rewetted)
+- Emissions|{Level-3 Species}|AFOLU|Land|Peat Burning:
```

Comment:
Is "Peat Burning" really the same as emissions from wetlands? Or do you want to explicitly remove emissions from wetlands?